### PR TITLE
[Compute Look Ahead]Add Newton information

### PR DESCRIPTION
**Pull request by rcaballeromx**

### CODE REVIEW

File: cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc [hunk 1]
```diff
@@ -1,24 +1,73 @@
...
+List of API endpoints that are no longer supported:
+
+* PUT, `{{/api/v<VERSION>/policies/firewall/network/container}}`
+* GET, `{{/api/v<VERSION>/policies/firewall/network}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/container/download}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/container}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/host/download}}`
+* GET, `{{/api/v<VERSION>/audits/firewall/network/host}}`
+
+|End of Support for Docker Access Control
```
Comment (Pubs-MV) [inline cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc]: @rcaballeromx , I think this has moved to Newton Update 1. @divyabhushan had sent a message, please confirm and update

File: cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc [hunk 2]
```diff
@@ -1,24 +1,73 @@
...
+|Docker Access Control (`defend/access/docker`) and the Access User role (`manage/authentication/roles`)  are no longer supported.
+
+|Support for Code Repo Scanning
+|Scanning your code repositories from the Prisma Cloud Compute console at *Compute > Monitor> Vulnerabilities > Code repositories* and use of Twistcli for code repo scanning was planned for End of Support in v32.0.xxx. The deprecation is now  extended until the next major release code named O'Neal (v33.0.xxx), when the support will be dropped.
```
Comment (Pubs-MV) [inline cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc]: End of Support is v32.00? Please confirm.

Newton is v31..00.xxx

O'Neal is v32.00.xx and not 33.0

File: cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc (hunk 1 above)
Comment (Pubs-MV) [reply cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc]: This is new blurb: 

Docker Access Control (Defend > Access > Docker) and Access User role (Manage > Authentication > Roles) were planned for End of Support in Newton (v31.00.xxx) as announced in [22.06 Release Notes](https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-release-notes/release-information/release-notes-22-06#:~:text=Upcoming%20Deprecation%20Notifications). The deprecation is now extended until the next release Newton Update 1 (v31.01.xxx), when the feature will be no longer supported .

Comment (rcaballeromx) [reply cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc]: Done.

**Pull request merged**